_data/projects/.net-compiler-platform-roslyn.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: .NET Compiler Platform ("Roslyn")
+desc: The .NET Compiler Platform ("Roslyn") provides open-source C# and Visual Basic
+  compilers with rich code analysis APIs.
+site: https://github.com/dotnet/roslyn
+tags:
+- roslyn
+- csx
+- script
+- compiler
+- C#
+- VB.NET
```

Comment:
@shiftkey Leave a comment on my gist if my code doesn't work anymore and you come up with something different ;)

I remember this being a huge PITA. :weary: